### DISCUSSION TITLE
Error al correr mis archivos en html

### DISCUSSION THREAD

**Discussion by CeViMu** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#318] [Q&A]
Hola compañeros, estoy tratando de correr mis archivos de html pero me aparece esto, alguien sabe como sollucionarlo?
<img width="442" alt="error1" src="https://user-images.githubusercontent.com/88991127/157489891-c1bc7435-d9ec-48fa-a353-4dc5e9e1ebd6.png">
<img width="442" alt="error" src="https://user-images.githubusercontent.com/88991127/157489893-5eca6441-9880-46b1-96c6-29a4fe2fe3bc.png">

**Comment by LeonardoRosalesG** [comment]
Deberías agregar en el URL el nombre de la pagina que quieres abrir.
Por ejemplo: localhost8080/index.html

Si usas Visual Code, hay una extensión llamada Live server. La puedes usar para desplegar un pequeño servidor en donde vivirá tu página.

**Reply by CeViMu** [reply]
Muchas gracias, voy a intentarlo.

**Comment by doguedogue** [comment]
Puedes darle clic desde el explorador de archivos y ver la página, algo asi:

![image](https://user-images.githubusercontent.com/23409026/157511622-773017e9-f72e-4de0-a740-b244418b74b4.png)

O bien, usa el servidor Live Server como extensión de visual code, es mucho mas fácil, solo le das clic derecho a tu página y le das Open with live server

![image](https://user-images.githubusercontent.com/23409026/157512082-1fe676d1-85df-461e-9ca7-5192302a91b7.png)

NOTA: Estás levantando tu página con algún servidor local el puerto 8080? (Ej. Tomcat)

**Comment by DanyVeneno** [comment]
Hola ceVimu para que no te compliques tanto con manejar un servidor virtual local aunque la herramienta que exponen los compañeros Explorers es efectiva
la otra en donde no te complicas tanto es :
1-le das doble click a tu archivo html y te abre con el navegador que tengas por defecto
2- si quieres escoger un navegador en especifico le das click derecho y te despliega un menu y te vas al link que dice abrir con y escoges el navegador de preferencia te recomiendo firefox developer bienespero te sirva pasa buena tarde

**Reply by CeViMu** [reply]
Muchas gracias, intentaré todas las opciones
